vignettes/ebola.Rmd

```diff
@@ -0,0 +1,615 @@
+
+```{r epred-sex-district, fig.cap="(ref:epred-sex-district)"}
+epred_draws_sex_district |>
+  dplyr::mutate(
+    sex = dplyr::case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = .epred, y = district)) +
+  tidybayes::stat_pointinterval() +
+  facet_grid(. ~ sex) +
+  labs(x = "Posterior expectation of the delay", y = "") +
+  theme_minimal()
+```
+
+## Linear predictor posteriors
+
+The `tidybayes` package also allows users to generate draws of the linear predictors for all distributional parameters using `add_linpred_draws`.
+For example, for the `mu` parameter in the sex-district stratified model:
+
+```{r}
+linpred_draws_sex_district <- obs_prep |>
+  as.data.frame() |>
+  modelr::data_grid(sex, district) |>
+  dplyr::mutate(obs_t = NA, pwindow_upr = NA, swindow_upr = NA) |>
+  add_linpred_draws(fit_sex_district, dpar = TRUE)
+```
+
+(ref:linpred-sex-district) The posterior distribution of the linear predictor of `mu` parameter within the sex-district stratified model.
+
+```{r linpred-sex-district, fig.cap="(ref:linpred-sex-district)"}
+linpred_draws_sex_district |>
+  dplyr::mutate(
+    sex = dplyr::case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = mu, y = district)) +
+  tidybayes::stat_pointinterval() +
+  facet_grid(. ~ sex) +
+  labs(x = "Posterior of the mu linear predictor", y = "") +
+  theme_minimal()
+```
+
+## Delay probability mass functions
+
+Posterior predictions of the delay distribution are an important output of an analysis with the `epidist` package.
+In this section, we demonstrate how to produce either a discrete probability mass function representation, or continuous probability density function representation of the delay distribution.
+
+### Discrete probability mass function
+
+The posterior predictive distribution under no truncation but daily censoring.
+
+No stratification:
+
+```{r}
+newdata_pmf <- obs_prep |>
+  as.data.frame() |>
+  dplyr::mutate(obs_t = 1000, pwindow_upr = 1, swindow_upr = 1)
+
+draws_pmf <- predicted_draws(fit, newdata = newdata_pmf, ndraws = 1000)
+
+ggplot(draws_pmf, aes(x = .prediction)) +
+  geom_bar(aes(y = after_stat(count / sum(count)))) +
+  labs(x = "PMF with daily censoring and no truncation", y = "") +
+  theme_minimal()
+```
+
+Stratification by sex:
+
+```{r}
+newdata_sex_pmf <- obs_prep |>
+  as.data.frame() |>
+  modelr::data_grid(sex) |>
+  dplyr::mutate(obs_t = 1000, pwindow_upr = 1, swindow_upr = 1)
+
+draws_sex_pmf <- predicted_draws(
+  fit_sex,
+  newdata = newdata_sex_pmf,
+  ndraws = 1000
+)
+
+draws_sex_pmf |>
+  dplyr::mutate(
+    sex = dplyr::case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = .prediction)) +
+  geom_bar(aes(y = after_stat(count / sum(count)))) +
+  labs(x = "PMF with daily censoring and no truncation", y = "") +
+  facet_grid(. ~ sex) +
+  theme_minimal()
+```
+
+Stratification by sex-district:
```

Comment:
This one looks broken right now. Again axis filter

Reply:
[4035227](https://github.com/epinowcast/epidist/pull/52/commits/40352278c509ec647c7255e233a6ffc6b6a57472)